examples/diff.c

```diff
@@ -335,3 +357,54 @@ static void diff_print_stats(git_diff *diff, struct opts *o)
 	git_buf_free(&b);
 	git_diff_stats_free(stats);
 }
+
+void die(char* message){
+	fprintf(stderr, message);
+	exit(EXIT_FAILURE);
+}
+
+struct pair extract_from_string(const char *cmd_opt)
+{
+	struct pair p;
+	char **endptr;
+
+
+	if (*cmd_opt=='/') {
```

Comment:
The slash is not only at the front. The format is `-B[<n>][/<m>]` and the explanation in the manpage mentions that these are two values handled differently. Not having the slash at the front doesn't mean that `m` should be the default. You can write `-B10%/20%` which this does not cover.